src/Tribe/Rewrite.php

```diff
@@ -10,6 +10,20 @@
  * Permalinks magic Happens over here!
  */
 class Tribe__Events__Rewrite extends Tribe__Rewrite {
+
+	/**
+	 * Constant holding the transient key for delayed triggered flush from activation.
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Line exceeds 80 characters; contains 86 characters

<sub>hash: d839e43a89956ce680255e6080380f8b</sub>